### DISCUSSION TITLE
ギミックシステム

### DISCUSSION THREAD

**Discussion by anminmakura** [TUSB/TheUnusualSkyBlock#92] [各種システムまとめ]
主にギミックの仕様ついてのまとめ。

**Comment by anminmakura** [comment]
### 無限奈落の進入ルートの現状の予定

TUSB神社に祭られている神様を怒らせ、神の使徒を召喚させる。
神の使徒はプレイヤーをデジョンさせるスキルを持っており、スキルを食らうことで無限奈落にいける。

### 以下神を怒らせる行為。

・12回以上鐘を鳴らす
・神社にスポーンしている池にいる魚を2匹以上倒す
・神社にスポーンしている羊を2匹以上倒す。
・定期的に自然に増える(上書きされる)お供えチェストの中身を空にする(逆にゴミを入れても怒らない優しさ)
・境内でスキル攻撃を12回以上行う。

**Comment by KumaCS** [comment]
スポナー破壊でディメンションにあわせたアイテムドロップ
ex)トカルトで通貨、クラウディアでエンパ、…
(ドロップ品については要検討)

**Comment by anminmakura** [comment]
スヤヤ氏案

裏劇場で観客がショーに介入してくるイベントを思いついた。
お助けアイテムとしてエリクサーを投げてきたり、ジェノサイドボムを投げてきたり。
ピンチだと観客が応援してきたりする。

**Comment by anminmakura** [comment]
ひかるつ氏案 まとめログ

ひかるつ

島のスポナーが壊されると
周囲のスポナーの湧く頻度が上がるとかよいのではと
もしくは敵が強くなるとか
難易度が高い島＝スポナーが多い上に質もある＝スポナーを全て破壊するまで気が抜けない＝最後まで緊張感がある


安眠まくら(-ヮ-)

壊したら難易度が上がる機構か


クマ

正直、アドベンチャーエリアじゃないと難しいかなって気もする
それこそ煉獄的な


スヤヤ

実装できるかどうか考えないで言うならここらへんかな
スポナーの湧き感覚が上がる
出てくる敵が強くなる
ボスが出る


ひかるつ

歴戦つすばーってのが何なのかわからないけど
ユニークなスポナーはちょっとぐらいあっても面白いんじゃないかなって思ったりする
例えば、アドベンチャーエリアに破壊できないスポナーがあるけど
ピストンを使って動力をあれこれするとブロックが引っこ抜かれて壊せるとか


安眠まくら(-ヮ-)

なるほど、面白い。
壊すとギミック作動のトラップスポナーほしい。
たぶん消滅条件で作動の応用でいけそう。
ギミックとしては擬態スポナーのスポナートロッコが消えた瞬間特定のfunctionイベントを呼び出すって奴。


ひかるつ

なんなら
今砂漠の宮殿で「壁」って書かれてあるとこも
スポナーを壁に埋め込んでおいて、
それを壊すと壁ごと壊れて先に進める仕組みとかできたらいいだろうなぁって思ってたりもした


安眠まくら(-ヮ-)

うんうん
予め消えるブロックを指定して配置して、
スポナーを壊したら周囲の消えるブロックを巻き込んで消滅するとかありだなって思った。


ナットボックス

壁が開いている間はファイアチャージ打てない、
壁が閉まっている間は打てる
ようにするとか


ひかるつ

ついでにこれも投下
https://twitter.com/TsuHikaru/status/1317787966735933440
![ひかるつアイディア](https://user-images.githubusercontent.com/39670701/107906819-704f7d80-6f95-11eb-9ead-fc4d82cd0894.png)



スヤヤ

3x3ブロックの壁にボタンがついていて
ボタンを押すと押したところが丸って表示されて
三目並べが開始されるギミックを思いついたので書いておく
負けるとリセットされる。


ひかるつ

HypixelのSkyblockのDungeonにそういうのあった気がする


スヤヤ

まんまそれ


ひかるつ

あれたのしいからすき


スヤヤ

負けてもリセットされるので何度でも挑めます


ナットボックス

謎解き入れるならゼルダ的なのが似合うと思うけど、
TUSBだとプレイヤーの肉体だけでは成立させにくいから中々難しいと思いましたまる
